lib.rs

```diff
@@ -326,7 +326,8 @@ impl WindowsResource {
     /// extern crate winresource;
     /// # use std::io;
     /// fn main() {
-    ///   if std::env::var("CARGO_CFG_TARGET_OS").unwrap() == "windows" {
```

Comment:
Actually, using `CARGO_CFG_TARGET_OS`, rather than `#[cfg(windows)]`  serves a purpose. See also #1 for a detailed explanation.

Reply:
Yeeah. This issue is a real bummer. There's just no good way to get the information of the _actual_ target OS for a `build.rs` in the cross-platform case, especially at compile-time. I've looked far and wide and turned up empty-handed.